sat/util.py

```diff
@@ -404,6 +404,11 @@ def get_resource_section_path(section):
             created
     """
     resource_path = os.path.join(os.environ['HOME'], '.config', 'sat', section)
+
+    sat_file = os.getenv("SAT_CONFIG_FILE")
+    if sat_file:
+        resource_path = os.path.join(os.path.dirname(os.environ["SAT_CONFIG_FILE"]), section)
```

Comment:
```suggestion
        resource_path = os.path.join(os.path.dirname(sat_file), section)
```
I feel you can use `sat_file` that you already retrieved from the environment rather than qurying again, this avoids key error also